docs/source/index.rst

```diff
@@ -118,12 +118,12 @@ with a :any:`Gaussian` covariance model.
 
 .. code-block:: python
 
-    from gstools import SRF, Gaussian
     import matplotlib.pyplot as plt
```

Comment:
matplotlib not used here.

Reply:
You're right! - I just deleted all the unnecessary imports.